# We want the escape/control characters in there, and not
# a backlash.
# Solaris-sepcific. Not for upstream.
--- tests/exp.right	2013-01-23 12:37:16.000000000 -0800
+++ tests/exp.right	2015-04-07 09:55:42.226643308 -0700
@@ -174,7 +174,7 @@
 [0;31m\]
 ]
 [hello
-\[\e[0;31m\]
+\[[0;31m\]
 
 hello
 eo
